MenuExample/Controller/MenuViewController.swift

```diff
@@ -19,6 +19,7 @@ class MenuViewController: UITableViewController {
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
 
+        self.view.backgroundColor = UIColor.clearColor()
```

Comment:
iOS9 for some reason ignore background color for this VC when it set from storyboard. With white background animation looks ugly...
Maybe it because iOS9 still in beta